docs/guide/overview.asciidoc

```diff
@@ -38,6 +36,69 @@ This data is buffered for a short period and sent on to APM Server.
 It listens on port 8200 by default and receives data from agents through a JSON HTTP API.
 Then it creates documents from that data and stores them in Elasticsearch.
 
-To visualize the data after it's sent to Elasticsearch,
-you can use the the dedicated APM UI bundled in X-Pack,
-or the pre-built open source Kibana dashboards that can be loaded directly via the {kibana-ref}/apm-getting-started.html[APM Kibana UI].
\ No newline at end of file
+*Elasticsearch* is a highly scalable open-source full-text search and analytics engine.
+It allows you to store, search, and analyze big volumes of data quickly and in near real time.
+Elasticsearch is used to store APM performance metrics and make use of its aggregations. 
+
+*Kibana* is an open source analytics and visualization platform designed to work with Elasticsearch.
+You use Kibana to search, view, and interact with data stored in Elasticsearch.
+You an use Kibana to visualize APM data by utilizing the dedicated APM UI bundled in Basic license,
+or the pre-built, open source,
+Kibana dashboards that can be loaded directly via the {kibana-ref}/apm-getting-started.html[APM Kibana UI].
+
+[[concepts]]
+[float]
+=== APM Concepts
+
+[float]
+==== Events
+APM agents capture different types of information from within their instrumented applications, known as events.
+Events can be <<errors,`Errors`>>, <<transaction-spans,`Spans`>>, or <<transactions,`Transactions`>>.
+These events are then streamed to a single endpoint in the APM Server which sorts and processes the events. 
+
+*<<errors,`Errors`>>* contain information about the original exception that occurred,
+or about a log created when the exception occurred.
+
+*<<transaction-spans,`Spans`>>* contain information about a specific code path that has been executed.
+They measure from the start to end of _something_,
+and they can have a parent/child relationship with other spans. 
+
+*<<transactions,`Transactions`>>* are a special kind of span that have metadata associated with them.
+You can think of transactions as the highest level of work you're measuring within a service.
+E.g., serving an HTTP request or running a specific background job.
+
+View the APM <<apm-data-model>> documentation for more information on Events. 
+
+[float]
+==== Component communication
+The APM Server is a {apm-server-ref}/why-separate-component.html[separate component by design] - it helps keep the agents light,
+prevents certain security risks, and improves compatibility across the Elastic and APM Stack.  
+
+The {apm-server-ref}/intake-api.html[Intake API] is what we call the internal protocol that APM agents use to talk to the APM Server. 
+After the APM Server has sorted and processed events from the APM agents (via the intake API),
```

Comment:
The APM Server doesn't sort event. You could change it to "validated", or if you like, just remove it.